src/vmm/src/lib.rs

```diff
@@ -1108,6 +1122,13 @@ impl Vmm {
             self.exit_evt = Some(exit_poll_evt_fd);
         }
 
+        #[cfg(target_arch = "aarch64")]
+        {
+            self.epoll_context
+                .add_epollin_event(self.exit_evt.as_ref().unwrap(), EpollDispatch::Exit)
+                .map_err(|_| StartMicrovmError::RegisterEvent)?;
+        }
+
```

Comment:
Please see my first comment. With that change in place I think this whole function can be simplified to look like this: 
```rust
fn register_events(&mut self) -> std::result::Result<(), StartMicrovmError> {
            self.epoll_context
                .add_epollin_event(self.exit_evt.as_ref().unwrap(), EpollDispatch::Exit)
                .map_err(|_| StartMicrovmError::RegisterEvent)?;
}
```

Reply:
Thanks. Let me try.

Reply:
Addressed in dc7eeff